### PR TITLE
Update README.md to follow the .NET pattern

### PR DESCRIPTION
**Pull request by kaerm**
Following the pattern in the .NET repo:  https://github.com/Azure/azure-storage-net/blob/master/README.md

### CODE REVIEW

**Comment by XiaoningLiu** [comment]
Hi @kaerm Thanks for contribution! This repo currently is in freeze state besides security issue fixes in favor of V12 series SDKs.

**Comment by kaerm** [comment]
That is exactly what I am trying to change. We have seen people landing on the page instead of v12 - my change is to make a notification about v12 more prominent @XiaoningLiu

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 This project provides a Node.js package and a browser compatible [JavaScript Client Library](https://github.com/Azure/azure-storage-node#azure-storage-javascript-client-library-for-browsers) that makes it easy to consume and manage Microsoft Azure Storage Services.
 
-> This README page is a reference to the SDK v2. For the new SDK v10, go to [Storage SDK v10 for JavaScript](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage).
+## If you would like to access our latest JS SDK, please refer to the Storage SDK v10 for JS link in the table below. If you would like more information on Azure's burgeoning effort to coordinate the development of the SDKs across services, of which this change is a part, please refer to [this article](https://azure.microsoft.com/en-us/blog/previewing-azure-sdks-following-new-azure-sdk-api-standards/).
```
Comment (ramya-rao-a) [inline README.md:10]: The v10 above should be v12

**Comment by ramya-rao-a** [comment]
@XiaoningLiu, @jiacfan,

I suggest that we get both #622 and #625 merged and release a patch update so that the readme on npm gets updated as well

**Pull request merged**